WordPress/playground-tools
# WIP: Update playground dependencies to v0.9.X

## What?

Let's update Playground dependencies to the latest version.

We may need to update other projects like Playground Plugin.



## Why?



It will fix WBEP images, support curl and other fixes features introduced in these versions.

## How?



- We need to update the dependencies and adapt the breaking changes to use the new functions.

## Testing Instructions






- Run `npx nx build wp-now && node dist/packages/wp-now/cli.js start`
- Observe Playground opens correctly in the browser.
